sdk/oracle/Azure.ResourceManager.OracleDatabase/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.1.0-beta.2
```

Comment:
remove this entry, use the one below and fill in the release date in the parentheses below